### ISSUE TITLE
sorry, I come again. I used the train and val set to train model, but the miou of test set is still low, just 0.54.  Can you tell me something about your training details？Thank you grateful !

### ISSUE THREAD

**Issue by qifengluo** [ydhongHIT/DDRNet#6]

**Comment by ydhongHIT** [comment]
All the training details are listed in the paper and I just follow the default settings of HRNet series. You get too big gaps between the val set and test set, which is not a normal phenomenon.

**Comment by ydhongHIT** [comment]
I merge val set into train set to form a new train set and use ohem.

**Comment by zptang1210** [comment]
I managed to add the model to a pytorch segmentation framework on github (just for the Cityscapes dataset). There are three things that we need to pay attention to:
1. It seems that the pretrained model (DDRNet23) used the MEAN and STD of Imagenet, instead of Cityscapes'.
2. It seems that the pretrained model is based on the original resolution of the image of the dataset without resizing or cropping.
3. The pretrained model set augment to True.

**Comment by ayhokuyan** [comment]
So you say we need to normalize according to the imagenet mean and std and also set augment to true when doing transfer learning right? ☺️
